migration/migrating_4_2_4/deploying-cam-4-2-4.adoc

```diff
@@ -1,72 +1,84 @@
 toc::[]
 
-You can install the {mtc-full} Operator on your {product-title} {product-version} target cluster and 4.2 source cluster. The {mtc-full} Operator installs the {mtc-full} ({mtc-short}) on the target cluster by default.
+You can install the {mtc-full} Operator on your {product-title} {product-version} target cluster and 4.2 source cluster.
 
-[NOTE]
+{mtc-short} is installed on the target cluster by default. You can install {mtc-short} link:https://access.redhat.com/articles/5064151[on an {product-title} 3 cluster or on a remote cluster].
+
+[IMPORTANT]
 ====
-Optional: You can configure the {mtc-full} Operator to install the {mtc-short} link:https://access.redhat.com/articles/5064151[on an {product-title} 3 cluster or on a remote cluster].
+You must install the same {mtc-short} version on all clusters.
 ====
 
-In a restricted environment, you can install the {mtc-full} Operator from a local mirror registry.
-
-After you have installed the {mtc-full} Operator on your clusters, you can launch the {mtc-short} web console.
+[id='installing-mtc_{context}']
+== Installing the {mtc-full} in a connected environment
 
-== Installing the {mtc-full} Operator
-
-You can install the {mtc-full} Operator with the Operator Lifecycle Manager on an {product-title} {product-version} target cluster and on an {product-title} 4.1 source cluster.
+You can install the {mtc-full} ({mtc-short}) in a connected environment.
 
 :!migrating-4-2-4:
 
 :context: target-4-2-4
 :target-4-2-4:
-include::modules/migration-installing-cam-operator-ocp-4.adoc[leveloffset=+2]
+include::modules/migration-installing-mtc-on-ocp-4.adoc[leveloffset=+2]
 :target-4-2-4!:
 
 :context: source-4-2-4
 :source-4-2-4:
-include::modules/migration-installing-cam-operator-ocp-4.adoc[leveloffset=+2]
+include::modules/migration-installing-mtc-on-ocp-4.adoc[leveloffset=+2]
 :source-4-2-4!:
 
 :context: disconnected-4-2-4
 :disconnected-4-2-4:
+[id='installing-mtc_{context}']
+== Installing the {mtc-full} in a restricted environment
 
-== Installing the {mtc-full} Operator in a restricted environment
-
-You can build a custom Operator catalog image for {product-title} 4, push it to a local mirror image registry, and configure the Operator Lifecycle Manager to install the Operator from the local registry.
+You can install the {mtc-full} ({mtc-short}) in a restricted environment.
 
-.Additional resources
-
-* xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
-
-[id='installing-cam-operator-prereq_{context}']
-=== Prerequisites
+[IMPORTANT]
+====
+You must install the same {mtc-short} version on all clusters.
+====
 
-* If you want to prune the default catalog and selectively mirror only a subset of Operators, install the xref:../../cli_reference/opm-cli.adoc#opm-cli[`opm` CLI].
+[id="using-operator-lifecycle-manager-on-restricted-networks_{context}"]
+=== Using Operator Lifecycle Manager on restricted networks
```

Comment:
I don't think you need this heading here. The paragraph that follows describe how to set up the OLM and environment to install the MTC. That isn't clear in the heading.